### PR TITLE
Add sorting feature to /admin/tags

### PR DESCRIPTION
**Pull request by Zhao-Andy**
## What type of PR is this? (check all applicable)
- [x] Refactor
- [x] Feature

## Description
This adds sorting options to `/admin/tags`, and also uses Ransack for all search / sorting related functions. And... that's pretty much it.

## Related Tickets & Documents
Closes https://github.com/forem/internalEngineering/issues/225

## QA Instructions, Screenshots, Recordings
1. Visit `localhost:3000/admin/tags` as an admin
2. Click on the headers, name, ID, alias for, and test that the sorting works
3. Click on Supported or Unsupported and see if sorting works there
4. Complete a search with the search function and try to sort your results

Screenshot of sorting arrows (Ransack default):
Screen Shot 2021-01-12 at 4 42 05 PM

### UI accessibility concerns?
The sort links should have aria labels for them. Also added some other aria labels for links. Not sure if I needed to add `#{tag.name} tag` to the tag links, but it seemed to make more sense to me?

## Added tests?
- [x] Yes

## Added to documentation?
- [x] No documentation needed

## [optional] Are there any post deployment tasks we need to perform?
Nope

## [optional] What gif best describes this PR or how it makes you feel?

A tiger mascot reminds you to stop and sort your recycling.

### CODE REVIEW

**Comment by Zhao-Andy** [comment]
@forem/design very small thing: let me know if you want me to change the default sorting arrow icons! They're the ones that came with the sorting gem we're using.

File: spec/system/admin/admin_views_tags_spec.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
```
Comment (Zhao-Andy) [inline spec/system/admin/admin_views_tags_spec.rb:1]: There's probably a better way to write this test, but IMO this is a good enough integration test that will probably be rewritten in Cypress.

**Review by jacobherrington — APPROVED** [review]

**Review by benhalpern — APPROVED** [review]

File: spec/system/admin/admin_views_tags_spec.rb (hunk 1 above)
Comment (lisasy) [reply spec/system/admin/admin_views_tags_spec.rb:1]: hello, can we reuse the icon we have on the Mod Panel?

https://share.getcloudapp.com/rRu0xmBp

**Review by rhymes — APPROVED** [review]
Looks cool!

**Comment by Zhao-Andy** [comment]
Heads up, gonna take a stab again at updating the sort icons before merging.

**Comment by Zhao-Andy** [comment]
Merging! Will try updating the design another time.

**Pull request merged**